README.md

```diff
@@ -103,7 +103,9 @@ spec:
 These alerts are generated from log data using LogQL queries processed by Loki. To create a log-based alert:
 
 - Ensure the `expr` field contains a valid LogQL query
-- Name the file with a `.logs.yaml` extension, this will render the following label `application.giantswarm.io/prometheus-rule-kind: loki` on the alert and ensure the alert is loaded into Loki.
+- Name the file with a `.logs.yaml` extension, this will render the following labels on the alert and ensure the alert is loaded into Loki:
+  * `application.giantswarm.io/prometheus-rule-kind: loki` (deprecated, will be removed once all management clusters have been upgraded to v30)
+  * `observability.giantswarm.io/rule-type: logs` (new label for releases > v30)
```

Comment:
Do we support both label for the time being ?

Reply:
Until all our MCs are on v30, this is ignored

Reply:
Ok :+1: